styleguide/components/31-molecules/tabs/tabs.bindings.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Don't see the added value of this file. How to initiate without the `data-tabs` attribute, is also documented in the documentation of allieTabs..

So this file can be removed as far as I'm concerned 🙂